src/meshapi/tasks.py

```diff
@@ -33,13 +39,30 @@ def run_update_panoramas() -> None:
         raise e
 
 
+@celery_app.task
+def run_update_from_uisp() -> None:
+    logging.info("Running UISP import & sync tasks")
+    try:
+        import_and_sync_uisp_devices(get_uisp_devices())
+        import_and_sync_uisp_links(get_uisp_links())
+        sync_link_table_into_los_objects()
+    except Exception as e:
+        # Make sure the failure gets logged.
+        logging.exception(e)
+        raise e
+
+
 celery_app.conf.beat_schedule = {
-    "run-database-backup-hourly": {
-        "task": "meshapi.tasks.run_database_backup",
-        "schedule": crontab(minute="0", hour="*/1"),
-    },
     "update-panoramas-hourly": {
         "task": "meshapi.tasks.run_update_panoramas",
         "schedule": crontab(minute="0", hour="*/1"),
     },
+    "import-from-uisp-hourly": {
+        "task": "meshapi.tasks.run_update_panoramas",
+        "schedule": crontab(minute="10", hour="*/1"),
+    },
+    "run-database-backup-hourly": {
+        "task": "meshapi.tasks.run_database_backup",
+        "schedule": crontab(minute="20", hour="*/1"),
+    },
```

Comment:
Good question. I don't know, so probably not. Adding a task to try to profile these UISP changes at the very least

Tell me more about the file touch thing, I'm not familiar

Reply:
Discussed on a call. Since there is not precise sequencing (or mutual exclusivity) required between these workers, we should be fine to leave this as it is, indicating a light preference for them to happen at different times (and for backups to happen last), but not enforcing it

I will verify that the data accesses in the UISP sync use `SELECT FOR UPDATE` in the right places, so that we don't have race conditions (against these other workers or against API/admin traffic)

Reply:
Patched a potential race condition, should be safe to run concurrently with anything now